app/models/docker_container_wizard_states/image.rb

```diff
@@ -4,8 +4,32 @@ class Image < ActiveRecord::Base
     belongs_to :wizard_state, :class_name => DockerContainerWizardState,
                               :foreign_key => :docker_container_wizard_state_id
     delegate :compute_resource_id, :to => :wizard_state
+    delegate :compute_resource, :to => :wizard_state
 
     validates :tag,             :presence => true
     validates :repository_name, :presence => true
+    validate :validate_image_exists
+
+    def name
+      "#{repository_name}:#{tag}"
+    end
+
+    def registry_name
+      @registry_name ||= if registry_id
+        DockerRegistry.find(registry_id).fetch(:name, _('Unnamed Registry'))
+      else
+        'Docker Hub'
+      end
+    end
+
+    def validate_image_exists
+      unless compute_resource.exist? name
+        error_msg = _("Container image %{image_name} could not be found on %{registry}.") % {
```

Comment:
When does this error message show up? I'm unable to see this error. Right now if I input a image that doesn't exist, and click on next, I get:

![screenshot from 2017-02-17 08-53-12](https://cloud.githubusercontent.com/assets/598891/23057044/9fe198d8-f4ee-11e6-9815-6e2b254a0a31.png)

If I try to write , I see this error (which is right, the image is not found in the compute resource):

![screenshot from 2017-02-17 08-55-22](https://cloud.githubusercontent.com/assets/598891/23057094/dad73524-f4ee-11e6-9ec6-fff2e1dcf946.png)

The error the issue describes should be displayed when the image is not available in the registry or the Docker host does not have it (e.g: if I had manually downloaded a 'fakenews:faketag' image to the Docker host, I should be able to move forward despite the image not being available, because it's on the Docker host)